### DISCUSSION TITLE
Issue with Academic verification for GitHub student developer pack.

### DISCUSSION THREAD

**Discussion by Shrishtytikariha** [community/community#127198] [GitHub Education]
### Hi everyone,

Hello GitHub Education Team,

I hope this message finds you well.

I am writing to bring to your attention an issue I have encountered while attempting to avail of the GitHub Student Developer Pack. Despite following the necessary steps, my academic verification has not been successful due to specific issues with my documentation.

Here are the details of the issue:

1. Two-Factor Authentication: I have enabled two-factor authentication on my GitHub account.
2. ID Card Issue: The ID card issued by my college does not explicitly mention the academic years (2024-2027), which seems to be causing the verification failure. However, this ID card is valid and officially issued by my institution.
3. Additional Documents: I have also attempted to upload other supporting documents, such as my Admission form ,but these were not accepted either.

I am attaching the following documents for your reference:

1. A screenshot of the issue encountered during the verification process.
2. A copy of my college ID card.
3.  A copy of  my Admission form.

I would greatly appreciate it if you could look into this matter and assist me in completing the verification process. The GitHub Student Developer Pack is crucial for my studies and projects, and resolving this issue promptly would be incredibly helpful.

Thank you for your attention and assistance.

Best regards,
Shrishty Tikariha
![Uploading Screenshot_2024-05-19-09-40-04-89_40deb401b9ffe8e1df2f1cc5ba480b12.jpg…]()
![IMG20240502135751](https://github.com/community/community/assets/168194445/d8d85331-cd6d-4430-bb78-44694e08ef3f)
![IMG20240501211953~2](https://github.com/community/community/assets/168194445/b6d4984d-a0f4-489c-ab9a-d4e10a40ae61)

**Comment by davevad93** [comment]
Hi @Shrishtytikariha ,

I think the main problem is that your documents lack a current date that GitHub needs as proof of your current enrollment. Initially, when I requested my student benefits, my application was rejected twice because the documents I've submitted only showed the academic year. I've been able to finally solve it by uploading a screenshot of my last exam results which showed a current date in a `DD/MM/YYYY` format. For example, if you're applying today, your document should display a recent date  as that would be a tangible proof of your current student status. If you can't provide such a document try contacting support by raising a ticket [here](https://support.github.com/contact/account) , select `Other` and explain your issue.
